packages/block-editor/src/hooks/color-panel.js

```diff
@@ -29,32 +30,22 @@ export default function ColorPanel( {
 	const definedColors = settings.filter( ( setting ) => setting?.colorValue );
 
 	useEffect( () => {
+		// If contrast checking is disabled, don't do anything.
 		if ( ! enableContrastChecking ) {
 			return;
 		}
-		if ( ! definedColors.length ) {
-			if ( detectedBackgroundColor ) {
-				setDetectedBackgroundColor();
-			}
-			if ( detectedColor ) {
-				setDetectedColor();
-			}
-			if ( detectedLinkColor ) {
-				setDetectedColor();
-			}
-			return;
-		}
 
-		if ( ! ref.current ) {
-			return;
-		}
```

Comment:
Why is this `ref.current` check removed? What happens if it's falsy and gets passed to `getComputedStyle`?

Reply:
Had a brainfart about how the ref.current would only be needed if there was no defined colors, but right now I can't recall the logic that was going on in my head at that moment, lol.